chia/plotters/plotters.py

```diff
@@ -0,0 +1,372 @@
+    POOLCONTRACT = 19
+    MADMAX_RMULTI2 = 20
+    BLADEBIT_WARMSTART = 21
+    BLADEBIT_NONUMA = 22
+    VERBOSE = 23
+    OVERRIDE_K = 25
+    ALT_FINGERPRINT = 26
+    EXCLUDE_FINAL_DIR = 27
+    CONNECT_TO_DAEMON = 28
+
+
+chia_plotter = [
+    Options.TMP_DIR,
+    Options.TMP_DIR2,
+    Options.FINAL_DIR,
+    Options.FILENAME,
+    Options.K,
+    Options.MEMO,
+    Options.ID,
+    Options.BUFF,
+    Options.NUM_BUCKETS,
+    Options.STRIPE_SIZE,
+    Options.NUM_THREADS,
+    Options.NOBITFIELD,
+    Options.OVERRIDE_K,
+    Options.ALT_FINGERPRINT,
+    Options.POOLCONTRACT,
+    Options.FARMERKEY,
+    Options.POOLKEY,
+    Options.PLOT_COUNT,
+    Options.EXCLUDE_FINAL_DIR,
+    Options.CONNECT_TO_DAEMON,
+]
+
+madmax_plotter = [
+    Options.K,
+    Options.PLOT_COUNT,
+    Options.NUM_THREADS,
+    Options.NUM_BUCKETS,
+    Options.MADMAX_NUM_BUCKETS_PHRASE3,
+    Options.TMP_DIR,
+    Options.TMP_DIR2,
+    Options.FINAL_DIR,
+    Options.MADMAX_WAITFORCOPY,
+    Options.POOLKEY,
+    Options.FARMERKEY,
+    Options.POOLCONTRACT,
+    Options.MADMAX_TMPTOGGLE,
+    Options.MADMAX_RMULTI2,
+]
+
+bladebit_plotter = [
+    Options.NUM_THREADS,
+    Options.PLOT_COUNT,
+    Options.FARMERKEY,
+    Options.POOLKEY,
+    Options.POOLCONTRACT,
+    Options.BLADEBIT_WARMSTART,
+    Options.BLADEBIT_NONUMA,
+    Options.FINAL_DIR,
+    Options.VERBOSE,
+]
+
+
+def get_plotters_root_path(root_path: Path) -> Path:
+    root_path = root_path.parent
+    return root_path / "plotters"
+
+
+def build_parser(subparsers, root_path, option_list, name, plotter_desc):
+    parser = subparsers.add_parser(name, description=plotter_desc)
+    for option in option_list:
+        if option is Options.K:
+            parser.add_argument(
+                "-k",
+                "--size",
+                type=int,
+                help="K value.",
+                default=32,
+            )
+        u_default = 0 if name == "chiapos" else 256
+        if option is Options.NUM_BUCKETS:
+            parser.add_argument(
+                "-u",
+                "--buckets",
+                type=int,
+                help="Number of buckets.",
+                default=u_default,
+            )
+        if option is Options.STRIPE_SIZE:
+            parser.add_argument(
+                "-s",
+                "--stripes",
+                type=int,
+                help="Stripe size.",
+                default=0,
+            )
+        if option is Options.TMP_DIR:
+            parser.add_argument(
+                "-t",
+                "--tmpdir",
+                type=str,
+                help="Temporary directory 1.",
+                default=str(root_path) + "/",
+            )
+        if option is Options.TMP_DIR2:
+            parser.add_argument(
+                "-2",
+                "--tmpdir2",
+                type=str,
+                help="Temporary directory 2.",
+                default=str(root_path) + "/",
+            )
```

Comment:
These match madMAx right now instead of matching `chia plots create`.  Is this what we want?  I don't particularly like the `_` separators in our options, but consistency with ourselves does seem valuable.

Reply:
Agreed that we should keep the options consistent with `chia plots create` to preserve functionality with existing user scripts/tools. So this would be changed to `--tmp2_dir` and we would also update `Options.FINAL_DIR` to `--final_dir`